colour/colorimetry/generation.py

```diff
@@ -0,0 +1,349 @@
+# -*- coding: utf-8 -*-
+"""
+Spectral Generation
+===================
+
+Defines various objects performing spectral generation:
+
+-   :func:`colour.constant_spd`
+-   :func:`colour.zeros_spd`
+-   :func:`colour.ones_spd`
+-   :func:`colour.gaussian_spd`
+-   :func:`colour.single_led_spd_Ohno2005`
+-   :func:`colour.multi_led_spd_Ohno2005`
+
+See Also
+--------
+`Spectrum Jupyter Notebook
+<http://nbviewer.jupyter.org/github/colour-science/colour-notebooks/\
+blob/master/notebooks/colorimetry/generation.ipynb>`_
+
+References
+----------
+-   :cite:`Ohno2005` : ﻿Ohno, Y. (2005). Spectral design considerations for
+    white LED color rendering. Optical Engineering, 44(11), 111302.
+    doi:10.1117/1.2130694
+-   :cite:`Ohno2008a` : Ohno, Y., & Davis, W. (2008). NIST CQS simulation 7.4.
+Retrieved from https://drive.google.com/file/d/\
```

Comment:
This link does not copy paste easily from source.

Reply:
Yeah unfortunately we don't (did not) have choice or it will break the .rst/.html output.